generator/feature_sorter.cpp

```diff
@@ -520,11 +522,12 @@ namespace feature
         }
       }
 
+      uint32_t featureId = numeric_limits<uint32_t>::max();
```

Comment:
Please, create a named constant (kInvalidFeatureId) in the header for the base class, and use it here.